cmd/ocm-backplane/upgrade/cmd.go

```diff
@@ -0,0 +1,55 @@
+package upgrade
```

Comment:
Rename this file to upgrade.go , to maintain the current structure of bp-cli

Reply:
done